<launch>
	<node pkg = "whud_stateos" type = "OSMain" name = "StateOS" output="screen">
		<param name="file_path" value="$(find whud_stateos)/config/Task"/>
		<param name="task_count" value="5"/>
	</node>
</launch>
